spec/memo_wise_spec.rb

```diff
@@ -133,6 +135,19 @@ def public_memowise_method
       public :public_memowise_method
 
       def unmemoized_method; end
+
+      def proc_method(proc)
```

Comment:
I don't think I see what this adds—to me this method is just the same as any other positional argument method. Thoughts?

Reply:
Yeah, was adding it mostly as documentation because I was unsure how explicit we want to be about the proc workaround. But can remove, totally agreed it's not actually testing anything new!